src/Chainweb/Pact/PactService/ExecBlock.hs

```diff
@@ -417,7 +417,7 @@ applyPactCmd isGenesis env miner txTimeLimit cmd = StateT $ \(T2 mcache maybeBlo
     T2 result mcache' <- do
       pd <- getTxContext (publicMetaOf gasLimitedCmd)
       if isGenesis
-      then liftIO $! applyGenesisCmd logger env P.noSPVSupport gasLimitedCmd
+      then liftIO $! applyGenesisCmd logger env P.noSPVSupport pd gasLimitedCmd
```

Comment:
correcting a historical wrong: `pd` should be `txCtx` or smth because i'ts not `PublicData` or `PublicMeta` anymore